packages/grafbase-urql-exchange/README.md

```diff
@@ -0,0 +1,89 @@
+# @grafbase/urql-exchange
+
+URQL-exchange for handling Server Sent Events (SSE)
+
+## Getting Started
+
+Follow these steps in a new or existing React application
+
+1. Install the dependencies
+
+```
+npm install urql graphql @grafbase/urql-exchange
+```
+
+2. Create URQL client
+
+```ts
+// client.ts
+import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'
+
+export const client = createClient({
+  url: process.env.GRAFBASE_API_URL,
+  fetchOptions: {
+    headers: {
+      authorization: `Bearer ${token}`
```

Comment:
Question: Do we read from `fetchOptions` inside the URQL exchange so we pass it along as a query param?

Reply:
Yes, `@grafbase/urql-exchange` accepts all headers and transforms them into query params

Reply:
Great. Thanks for clarifying.